common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2406,7 +2406,7 @@ object CelebornConf extends Logging {
       .version("0.3.0")
       .doc("Intervals between device monitor to check disk.")
       .timeConf(TimeUnit.MILLISECONDS)
-      .createWithDefaultString("60s")
+      .createWithDefaultString("30s")
```

Comment:
let's add this change to `docs/migration.md`.

Reply:
update